src/main/java/sirius/biz/tenants/jdbc/SQLUserAccount.java

```diff
@@ -37,15 +37,10 @@
 @TranslationSource(UserAccount.class)
 public class SQLUserAccount extends SQLTenantAware implements UserAccount<Long, SQLTenant> {
 
-    public static final Mapping USER_ACCOUNT_DATA = Mapping.named("userAccountData");
```

Comment:
in S2 its used in some functions like admin-support

Reply:
this was redundant as the constant is already defined in the interface UserAccount